### PR TITLE
most of website now has a layout

### PR DESCRIPTION
**Pull request by Noviny**
@alexreardon Got a lot of work done on this, but ran into an issue with codesandboxer not parsing `.jsx` files when I was trying to add it in to the examples pages.

Should probably talk about the examples as well, as they don't work very well in the space this version of the site provides them.

My next step is update codesandboxer, and reconvene with you to talk about examples.

### CODE REVIEW

File: website/gatsby-node.js [hunk 1]
```diff
@@ -133,3 +137,19 @@ exports.createPages = ({ graphql, boundActionCreators }/* : NodeParams */)/* : P
     );
   });
 };
+
+exports.onCreatePage = async ({ page, boundActionCreators }) => {
+  const { createPage } = boundActionCreators;
+
+  return new Promise((resolve, reject) => {
```
Comment (alexreardon) [inline website/gatsby-node.js]: kewl! Does this still work well with SSR?

Comment (alexreardon) [reply website/gatsby-node.js]: Does it need to be a promise?

**Comment by alexreardon** [comment]
I am keen to iterate on this. Let's get the build passing an merge it in

File: src/state/move-to-next-index/in-home-list.js [hunk 2]
```diff
@@ -4,7 +4,6 @@ import getDraggablesInsideDroppable from '../get-draggables-inside-droppable';
 import { patch, subtract } from '../position';
 import withDroppableDisplacement from '../with-droppable-displacement';
 import isTotallyVisibleInNewLocation from './is-totally-visible-in-new-location';
-// import getScrollJumpResult from './get-scroll-jump-result';
```
[the remark this answers is not in the document]
Comment (Noviny) [reply src/state/move-to-next-index/in-home-list.js]: Codesandboxer by default was attempting to add this file to the bundle, which was causing a fairly large break. Preferably, this file itself shouldn't be uploaded. Was this as commented out important to maintain?

File: website/gatsby-node.js (hunk 1 above)
Comment (Noviny) [reply website/gatsby-node.js]: This is the recommended way to do this from gatsby's docs. I don't think it should hit SSR, as this is run when gatsby is generating the site's data.

**Comment by Noviny** [comment]
I'm pretty confused. The build errors here are to do with unused variables that I removed on my latest PR.

File: src/state/move-to-next-index/in-home-list.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (alexreardon) [reply src/state/move-to-next-index/in-home-list.js]: no idea why it is just commented..

File: website/src/components/Sidebar/index.jsx [hunk 3]
```diff
@@ -0,0 +1,100 @@
+// @flow
+import React, { Fragment } from 'react';
+import Link from 'gatsby-link';
+import styled, { css } from 'styled-components';
+import { colors as akColors } from '@atlaskit/theme';
+import { colors } from '../../constants';
+import type { docsPage, sitePage } from '../types';
+import { getTitleFromExamplePath } from '../../utils';
+
+const Sidebar = styled.div`
```
Comment (alexreardon) [inline website/src/components/Sidebar/index.jsx]: I would ask that these be based on the grid, but given that i plan on moving the sidebar out we can leave it like this for now

**Comment by alexreardon** [comment]
Maybe pull in master?

**Comment by alexreardon** [comment]
If that fails, perhaps recreate the pr

**Pull request merged**